pkgs/redis/client.go

```diff
@@ -79,25 +78,13 @@ func Get(ctx context.Context, key string) (string, error) {
 	return val, nil
 }
 
-func Set(ctx context.Context, key, value string, expiration time.Duration) error {
-	return RedisClient.Set(ctx, key, value, expiration).Err()
+func Set(ctx context.Context, key, value string) error {
+	return RedisClient.Set(ctx, key, value, 0).Err()
```

Comment:
Any reason for this change? Shouldn't we keep the expiry param?

Reply:
But why use a separate function when one can do the job?

Reply:
In this case, `Set()` had been doing the job well and good for setting with expiries as well. So yeah no need for a separate helper in here.